File: November/Day11/26_Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
         int k, i=1;
        for(k=1;k<nums.size();k++)
        {
          if(nums[k-1]!=nums[k])
          {
              nums[i] = nums[k];
              i++;
          }
        }
        return i;
        
    }
};

File: November/Day11/26_Remove_Duplicates_from_Sorted_Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int point=0;
        for(int i=0;i<nums.length-1;i++){
            if(nums[i]!=nums[i+1]){
                nums[++point]=nums[i+1];
            }
        }
        return point+1;
    }
}
